lib/WasmReader/WasmSignature.cpp

```diff
@@ -79,11 +79,14 @@ size_t WasmSignature::GetShortSig() const
     return m_shortSig;
 }
 
+template<bool useShortSig>
 bool WasmSignature::IsEquivalent(const WasmSignature* sig) const
```

Comment:
a default parameter `useShortSig` might be a bit simpler?

Reply:
I did, the default value is in the header